contrib/platform/test/com/sun/jna/platform/win32/User32WindowMessagesTest.java

```diff
@@ -313,7 +313,7 @@ public CallBackFindWindowHandleByWindowclass(String windowClass) {
         public boolean callback(HWND hWnd, Pointer data) {
 
             char[] windowText = new char[512];
-            User32.INSTANCE.GetClassName(hWnd, windowText, 512);
+            assertCallSucceeded("GetClassName", User32.INSTANCE.GetClassName(hWnd, windowText, windowText.length) != 0);
```

Comment:
Is there supposed to be a new method here?   Similar comment to another PR, what's the benefit of this vs. `assertEquals("GetClassName failed", WinErrors.ERROR_SUCCESS, User32....);` ?

Reply:
Hmm. This is not a new method? I'm simply using the method exposed by the base class that gives the nice error message in case the call to GetClassName failed.

Reply:
Ah, sorry, I missed that it was inherited.